packages/mermaid/src/diagrams/flowchart/flowRenderer-v3-unified.ts

```diff
@@ -54,7 +40,7 @@ export const draw = async function (text: string, id: string, _version: string,
 
   data4Layout.type = diag.type;
   data4Layout.layoutAlgorithm = layout;
-  data4Layout.direction = DIR;
+  data4Layout.direction = getDirection();
```

Comment:
Even though this makes it good for clean code, sometimes it makes it hard to debug without the variable.